src/firebolt/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.9.2"
+__version__ = "0.9.3"
```

Comment:
This does not need to be updated manually, we have a GH action that does it.

Reply:
Yeah, I mentioned this in Slack https://firebolt-analytics.slack.com/archives/C03748V9JK0/p1660675364408029. As a short-term solution, I had to make dbt install the SDK from a specific branch on github. The only way I could be certain this was happenning correctly was to bump the version number. I'll decrement it again when it's ready to merge.

Which, it's not because I'm still battling the unit tests.